tools/contractdeployer/contract_deployer.go

```diff
@@ -250,7 +250,7 @@ func getContractCode(cfg *Config) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
-		address := common.BytesToAddress(common.Hex2Bytes(cfg.ConstructorParams[1]))
+		address := common.HexToAddress(cfg.ConstructorParams[1])
```

Comment:
so weird. they look equivalent

Reply:
yeah I know, I've done that before too. Those utils are bizarre, like some of them will expect 0x, some won't and stuff I think.